Common/GPU/Vulkan/VulkanRenderManager.cpp

```diff
@@ -1462,6 +1477,11 @@ void VulkanRenderManager::Run(VKRRenderThreadTask &task) {
 	}
 	frameData.SubmitPending(vulkan_, FrameSubmitType::Pending, frameDataShared_);
 
+	// Flush descriptors.
+	double descStart = time_now_d();
+	FlushDescriptors(task.frame);
```

Comment:
In theory, we could create these on a separate thread even, and wait for them, somewhat like shaders (of course, they'd need to be continuous, so this would only help if it could complete between the time drawing was enqueued and when the frame was picked up by the renderer.)

Not sure if the thread waits would definitely be worth it, but if this was a decent win, most devices nowadays have more cores than we'll ever be able to effectively use (I've been meaning to move atrac to a thread, for this reason - even if it only helps a tiny bit, I think it should help... does actually show up in profiles these days.)

-[Unknown]

Reply:
Yep, I thought about that, but it doesn't really seem necessary - there's usually plenty of time left to do these on the render thread without slowing anything down, even on really slow devices. But it might be worth experimenting with - the more parallelism, the more this crowd of CPUs can clock down...